docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 15 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
```

Comment:
Hi Jake, just spotted that the metadata needs to be tweaked. The objectives are now part of the metadata, here's an example from the latest VS Code Guide:
```
---
layout: guide
summary_title: "Codewind in VS Code"
title: "Getting Started with Codewind in VS Code"
categories: guides
description: "Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use."
permalink: codewind-vscode-quick-guide.html
duration: 5 minutes
`keywords: Codewind, VS Code, microservice`
`objectives: ["Install Visual Studio Code (VS Code) and Codewind.", "Develop a simple microservice that uses Eclipse Codewind in VS Code."]`
`icon: images/learn/icon_logoVScode.svg`
---
```
So change `guide_description:` to `description:`
And add:
```
keywords: Codewind, CodeReady, Workspace
objectives: ["Install CodeReady Workspaces and Codewind.", "Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces."]
icon: images/learn/icon_logoVScode.svg    
```

For the `objectives:` keep the syntax exactly as I've typed above including the `[` and `]` .
For the icon image, we need a Codeready Workspaces icon.